WordPress/Classes/Extensions/UIAlertController+Helpers.swift

```diff
@@ -16,7 +16,42 @@ extension UIAlertController {
         while leafViewController.presentedViewController != nil && !leafViewController.presentedViewController!.isBeingDismissed {
             leafViewController = leafViewController.presentedViewController!
         }
-
         leafViewController.present(self, animated: true, completion: nil)
     }
 }
+
+//MARK: - copy text to Clipboard
+extension UIAlertController {
+    /// This method is used for presenting the Action sheet
+    /// for copying text to clipboard. The action sheet has 2 options:
+    /// copy: will copy the text to the clipboard
+    /// cancel: dismiss the action sheet
+    @objc static func copyTextAlertController(_ text: String?,
+                                              completion: ((Bool) -> Void)? = nil) -> UIAlertController? {
+        guard let text = text else {
+            completion?(false)
+            return nil
+        }
+        let alertController = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        alertController.addDefaultActionWithTitle(NSLocalizedString("Copy", comment: "Copy button")) { _ in
+            UIPasteboard.general.string = text
```

Comment:
`UIPasteboard` has a `url` property that seems would semantically fit better here than `string` here — I've never used the Pasteboard API's, so I have no idea if they have some weird behaviours, but seems perfect for the job here.